bitnami/elasticsearch/README.md

```diff
@@ -141,7 +144,11 @@ The following table lists the configurable parameters of the Elasticsearch chart
 | `metrics.image.pullPolicy`                        | Metrics exporter image pull policy                                                                                        | `Always`                                                            |
 | `metrics.service.type`                            | Metrics exporter endpoint service type                                                                                    | `ClusterIP`                                                         |
 | `metrics.resources`                               | Metrics exporter resource requests/limit                                                                                  | `requests: { cpu: "25m" }`                                          |
-
+| `metrics.name`                                    | Metrics pod name                                                                                                          | `metrics`                                                           |
```

Comment:
delete the `metrics.name` row, I think it is duplicated

Reply:
Done